app/fetch.py

```diff
@@ -132,7 +132,7 @@ def add_desc(d: Any, download_url: str) -> None:
     data = await get_content_cached(files_url, timeout=REQUEST_TIMEOUT)
 
     with io.BytesIO(data) as f:
-        with tarfile.open(fileobj=f, mode="r:gz") as tar:
+        with ExtTarFile.open(fileobj=f, mode="r") as tar:
```

Comment:
Does this work?  I thought you had to have mode `r:*` for auto-decompression.

Reply:
r == r:*